### ISSUE TITLE
Provide an option to always set defined environment variable regardless of how the config was specified

### ISSUE THREAD

**Issue by alallier** [rooseveltframework/source-configs#245]
Provide an option to always set defined environment variable regardless of how the config was specified. This is important because some modules only take config via Environment variables.

Imagine a schema like this:

**configSchema.js**
```js
{
  database: {
    hostname: {
      envVar: 'POSTGRES_DB_HOST',
      desc: 'Postgres host url',
      default: 'localhost'
    },
    port: {
      envVar: 'POSTGRES_DB_PORT',
      desc: 'Postgres port',
      default: 5432
    }
  }
}
```

Per how source configs is designed you can provide configuration (as specified in the [README](https://github.com/rooseveltframework/source-configs/blob/main/README.md?plain=1#L8-L16)) from command line arguments, environment variables, deployment config file, and defaults.

So let's say for the database hostname the configuration is defined by a command line argument and the environment variable `POSTGRES_DB_HOST` ends up not being defined. Certain modules can only take configuration from defined environment variables.

I propose we add a config item to the schema, like `alwaysDefineEnv`, (like below) that will always define the environment variable regardless of how the configuration is defined. So in the case above defining the database host from argument would also set the defined environment variable

Proposed new **configSchema.js**
```js
{
  database: {
    hostname: {
      envVar: 'POSTGRES_DB_HOST',
      desc: 'Postgres host url',
      default: 'localhost',
      alwaysDefineEnv: true
    },
    port: {
      envVar: 'POSTGRES_DB_PORT',
      desc: 'Postgres port',
      default: 5432
    }
  }
}
```

TLDR; Add configs option that always set the specified ENV if it exists, regardless of how the config item is set.

**Comment by alallier** [comment]
It looks like you can, https://stackoverflow.com/a/10829666/5223447, I haven't tested it though. Something my team has on the backlog

**Comment by kethinov** [comment]
Ah yeah — you're correct. I thought you were referring to setting env vars for the whole system, not just for the node process and its subprocesses.

In any event, I'm not sure a feature like this makes sense to have in this module. `source-configs` is about getting configs from the environment, not altering the environment or configuring downstream dependencies.
